osd/recover_master_node_is_required.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Cluster has not sufficient number of master nodes",
+    "description" : "SRE have noticed a change in the state of one or more of the cluster's master nodes. Please note that if you change the running state of one of the master nodes manually then SRE won't be able to support your cluster anymore. We suggest you to revert any changes made on master instances.",
```

Comment:
> SRE won't be able to support your cluster anymore

we never want to tell a customer that we won't support their cluster anymore. we should, instead, note that this may impact the stability of the cluster and that the cluster's SLA may also be impacted